Makefile

```diff
@@ -18,7 +18,11 @@ TIMEOUT ?= 14400s
 # TEST_EXEC_NODES=1, otherwise by default the specs are run in parallel on 4 ginkgo test node.
 # NOTE: Any TEST_EXEC_NODES value greater than one runs the spec in parallel
 # on the same number of ginkgo test nodes.
-TEST_EXEC_NODES ?= 4
+if $(CLUSTER_ON_IBM_CLOUD)
+   TEST_EXEC_NODES ?= 24
```

Comment:
Are you sure of using 24 virtual ginkgo nodes? You are not confusing this with the Kubernetes cluster nodes, right?

Reply:
yes, the execution in the IBM Cloud cluster is completed very fast so it does not impact ginkgo

Reply:
@rnapoles-rh the unit tests are failing with `Makefile:21: *** missing separator.  Stop.` have you checked it on local?
And I am not able to understand the `if` condition here. What value are we supposed to assign to the env  `CLUSTER_ON_IBM_CLOUD`?

Reply:
> And I am not able to understand the if condition here. What value are we supposed to assign to the env CLUSTER_ON_IBM_CLOUD?

@anandrkskd AFAIU we are assigning `true` to `CLUSTER_ON_IBM_CLOUD` for IBM cloud. `ifeq` checks whether two values are same or not. @rnapoles-rh correct me if I am wrong.

Reply:
> > And I am not able to understand the if condition here. What value are we supposed to assign to the env CLUSTER_ON_IBM_CLOUD?
> 
> @anandrkskd AFAIU we are assigning `true` to `CLUSTER_ON_IBM_CLOUD` for IBM cloud. `ifeq` checks whether two values are same or not. @rnapoles-rh correct me if I am wrong.

Yes, it is set in the Jenkins script as:
`--env "CLUSTER_ON_IBM_CLOUD=true"`

Reply:
> @rnapoles-rh the unit tests are failing with `Makefile:21: *** missing separator. Stop.` have you checked it on local?
> And I am not able to understand the `if` condition here. What value are we supposed to assign to the env `CLUSTER_ON_IBM_CLOUD`?

Yes, I had a syntax error which is now fixed

Reply:
> > @rnapoles-rh the unit tests are failing with `Makefile:21: *** missing separator. Stop.` have you checked it on local?
> > And I am not able to understand the `if` condition here. What value are we supposed to assign to the env `CLUSTER_ON_IBM_CLOUD`?
> 
> Yes, I had a syntax error which is now fixed

@rnapoles-rh Its still failing on the same line 
```
[ssh:Fedora-32-minikube] Makefile:21: *** missing separator.  Stop.
[ssh:Fedora-32-minikube] !!!run failed, skipping!!!
2021/09/02 20:12:25 : failed due to err Failed the test, see logs above ^
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-09-02T20:12:25Z"}
error: failed to execute wrapped command: exit status 1
```

Removing lgtm until it gets fixed.

/lgtm cancel